moped-database/migrations/1682011649641_create_new_subcomponents/up.sql

```diff
@@ -0,0 +1,24 @@
+INSERT INTO moped_subcomponents (subcomponent_name) values
+    ('Continuous flow intersection'),
+    ('Curb extension'),
+    ('Detection - Bicycle'),
+    ('Vehicle lane reconfiguration'),
+    ('Protected intersection - Full'),
+    ('Protected intersection - Full - Shared use'),
+    ('Protected intersection - partial'),
```

Comment:
if the F in Full is capitalized, should the P here be capitalized too?

Reply:
thank you! i'm going to cross check all these against the access DB mappings and fix them for consistency.

Reply:
I fixed this—thank you